Tests/SnapshotTests/Components/SegmentedSwitchTests.swift

```diff
@@ -0,0 +1,9 @@
+import XCTest
+@testable import Orbit
+
+class SegmentedSwitchTests: SnapshotTestCase {
+
+    func testSegmentedSwitch() {
+        assert(SegmentedSwitchPreviews.snapshot)
```

Comment:
The snapshot reveal some UI issues

- rounded corners have different sizes
- the separator crosses the border

<img width="308" alt="image" src="https://user-images.githubusercontent.com/35844477/209155691-ece7ae7b-31a4-496f-bd06-a027e57e3081.png">

Reply:
Extra elevation was added for state with no selection. Fixed

Reply:
Even after the change, the border radius of inner selected view is equal to the outer one - it needs to be smaller to visually match. See 

```
.clipShape(RoundedRectangle(cornerRadius: BorderRadius.default - 1))
```

Reply:
You're right, I haven't check carefully properties in figma. The outer one is 6pts, while inner one is 5. Updated